test/setups/complex_shape.jl

```diff
@@ -1,8 +1,9 @@
-@testset verbose=true "Complex Shapes" begin
+@testset verbose=true "ComplexShapes" begin
```

Comment:
```suggestion
@testset verbose=true "ComplexShape" begin
```
I meant to use the struct name.